src/midi/midi_handler.h

```diff
@@ -26,7 +26,8 @@
 
 #include <cstdint>
 
-#define SYSEX_SIZE 8192
+constexpr auto MIDI_MSG_SIZE = 8;
+constexpr auto MIDI_SYSEX_SIZE = 8192;
```

Comment:
I think MIDI_MSG_SIZE does not need to be exposed.

MIDI_SYSEX_SIZE - maybe it should be transferred from midi_handler.h to midi.h? (I'm not sure).

Reply:
Yes, if we try a three-byte message length, then the concept of MSG_SIZE being a constant magic number goes away.  In which case the need to refactor the SYSEX_SIZE as a "sister-value" also goes away in this PR.. unless we want to include it still, which I'm happy doing. 

I know you want to do some heavier MIDI refactoring with the handlers anyway, before we take on the sf2-load-in-parallel PR. So maybe can do this in the refactoring PR?

Reply:
> maybe it should be transferred from midi_handler.h to midi.h

I think so too.
- `MIDI_SYSEX_SIZE` defines a size for a struct defined in `midi.cpp`, so logically it should live in either the cpp or header
- `MIDI_SYSEX_SIZE` is used by other classes that include `midi.h`, so needs to go up in the header.